### ISSUE TITLE
Custom theme documentation is incomplete

### ISSUE THREAD

**Issue by avalonv** [tridactyl/tridactyl#1786]
Tridactyl version: 1.16.2pre2999-95db0a8
Firefox version: Mozilla Firefox 68.0.1
Operating system: linux

Attempting to set a custom theme fails silently and sets the default theme instead, no errors are shown.

Additionally, running `:get theme` displays the name of the custom theme I tried to set, despite it actually being the default theme.

**Comment by glacambre** [comment]
`:get theme` shouldn't display the name your theme when you're trying to set a custom theme (it should display `customtheme` instead). You need to use `:colourscheme` when you want to load a theme from disk.

**Comment by avalonv** [comment]
To clarify, I have a theme called "quake2.css" in my `~/.config/tridactyl/themes/` folder, which is just a copy of quake.css for testing purposes.

Attempting to set it with `:colourscheme quake2` fails with no messages being shown, and the default theme is applied instead. Attempting to set a theme that doesn't exist like `:colourscheme fake` produces an error as expected.

> :get theme shouldn't display the name your theme when you're trying to set a custom theme

Sorry, I don't understand this sentence, could you rephrase it? Running `:get theme` displays `# theme = quake2` for me.

**Comment by glacambre** [comment]
>Sorry, I don't understand this sentence, could you rephrase it? Running :get theme displays # theme = quake2 for me.

Yeah, I thought it shouldn't do that (I thought you'd get a special value that would let tridactyl know it should find your theme elsewhere). I was remembering how custom themes worked correctly though and having `theme` set to `quake2` is normal and should work.

Could you run `:get customthemes` and see if it contains a `quake2` entry the value of which is the content of your `$HOME/.config/tridactyl/themes/quake2.css` file? And if it does, did you prefix your selectors with `.TridactylThemeQuake2`?

**Comment by bovine3dom** [comment]
We should probably document all the hoops one must jump through to make a custom theme on the Wiki, and improve the error messages if no theme is found.

**Comment by glacambre** [comment]
I think it'd be better to add this information to `:colourscheme`'s documentation. The only thing really missing is that if you want to apply your theme to Tridactyl elements without affecting the page, you should prefix every selector with `.TridactylTheme${CapitalizedThemeName}`.

**Comment by avalonv** [comment]
> And if it does, did you prefix your css selectors with .TridactylThemeQuake2?

There we go, doing that fixed it. Thanks so much

**Comment by glacambre** [comment]
I'll re-open this until we update the documentation, you're not the first who tripped on this :)

**Comment by bovine3dom** [comment]
I've made themes a lot simpler - no need for the weird classnames - but the documentation still needs a lot of work so I'll leave this open for now

**Comment by bovine3dom** [comment]
`:colourscheme` is pretty well documented now i think
